asmrun/signals_osdep.h

```diff
@@ -287,7 +307,7 @@
 
 /****************** PowerPC, ELF (Linux) */
 
-#elif defined(TARGET_power) && defined(SYS_elf)
+#elif defined(TARGET_power) && defined(SYS_elf) && !defined(__NetBSD__)
```

Comment:
Why do you need this if you set `system=bsd_elf` in configure? Does a system configured this way still define `SYS_elf`? (I'm not familiar with this part of the configuration logic, and would have assumed that only `SYS_bsd_self` was defined).

Reply:
This is a remnant from when I tried to use the existing setting `system=elf`, I'll clean this up.